### PR TITLE
Add end-to-end tutorial for EgoBipartiteGraphSAGE model.

### PR DESCRIPTION
**Pull request by Seventeen17**
1. Trained with GraphLearn-Training: implement model with u2i2i and
   i2i2i meta-path;
2. Infered user embedding with Dynamic-Graph-Service:
   - Add getQuery http request;
   - Refine EgoGraph/EgoTensor for parsing sampled fbs records;
   - Refine App and add uts.

### CODE REVIEW

File: docs/en/dgs/tutorial.md [hunk 1]
```diff
@@ -6,13 +6,15 @@ This document is an e2e tutorial of offline training and online inference for a
   - Tensorflow model serving.
 
 Here is an example of a supervised job with EgoBipartiteSage, containing the following sections.
-1. Prepare data, including bulk-loading data for offline training and streaming data for online inference.
-2. Train the EgoBipartiteSage model using the offline bulk-loading data.
-3. Exporting TF model.
-4. Deploying TF model on tensorflow model serving.
+1. Prepare u2i and i2i bipartite graph data, including bulk-loading data for offline training and streaming data for online inference.
+2. Train the EgoBipartiteSage model using the offline bulk-loading data, it
+contain user model and item model.
```
Comment (baoleai) [inline docs/en/dgs/tutorial.md:11]: typo

File: docs/en/dgs/tutorial.md [hunk 2]
```diff
@@ -6,13 +6,15 @@ This document is an e2e tutorial of offline training and online inference for a
...
+3. Exporting user model.
+4. Deploying the model on tensorflow model serving.
 5. Deploy the online dynamic graph service and ingest the streaming data.
 6. Start Java Client, sample and predict.
 
+The first 3 part need to to use the training framework GraphLearn-Training, a detailed description of the reference [GraphLearn-Training](../gl/intro.md).
```
Comment (baoleai) [inline docs/en/dgs/tutorial.md:17]: typo

File: docs/en/dgs/tutorial.md [hunk 3]
```diff
@@ -33,81 +35,53 @@ The generated data are stored in `/tmp/u2i_gen/training` and `/tmp/u2i_gen/strea
...
-FIXME(@Seventeen17): refactor this section
 
 First, export model as tf SavedModel, we need to filter some of the placeholders as model serving inputs based on the
 computational graph, you can view the computational graph with the help of Tensorboard to determine the inputs for the
 serving subgraph.
 
-The offline training model is saved in `graphlearn/examples/tf/ego_sage/ckpt`, and we save the final serving model in
-`./ego_sage_sup_model` directory, the inputs to the subgraphs are the placeholders `0,2,3`.
+The offline training model is saved in `graphlearn/examples/tf/ego_bipartite_sage/ckpt`, and we save the final serving model in
+`graphlearn/examples/tf/serving/ego_bipartite_sage` directory, the inputs to the user-subgraph are the placeholders `0,3,4` according to TensorBoard.
```
Comment (baoleai) [inline docs/en/dgs/tutorial.md:54]: we should consider better apporach.

File: dynamic_graph_service/gsl_client/src/main/java/org/aliyun/gsl_client/Query.java [hunk 4]
```diff
@@ -29,9 +29,9 @@ public void setId(String id) {
     this.id = id;
   }
 
-  public void feed(String alias, DataSource source) throws UserException {
-    PlanNode node = plan.getNode(alias);
-    node.setSource(source);
+  public void feed(DataSource source) throws UserException {
```
Comment (baoleai) [inline dynamic_graph_service/gsl_client/src/main/java/org/aliyun/gsl_client/Query.java:32]: why del alias?

Comment (Seventeen17) [reply dynamic_graph_service/gsl_client/src/main/java/org/aliyun/gsl_client/Query.java:32]: We only support feed input for source op now.

File: docs/en/dgs/tutorial.md (hunk 3 above)
Comment (Seventeen17) [reply docs/en/dgs/tutorial.md:54]: Yes, but for now on, I haven't find a more appropriate solution, I will provide more automated tool in subsequent commits.

**Review by goldenleaves — APPROVED** [review]
LGTM

**Review by LiSu — APPROVED** [review]

**Pull request merged**